docs/integration.md

```diff
@@ -0,0 +1,435 @@
+
+This page explains how to package Chutney for use in real Development or QA environments.
+
+As Chutney server is a [Spring Boot](https://docs.spring.io/spring-boot/docs/2.7.x/reference/html/index.html) application, only an all-in-one Chutney [Spring Boot executable jar](https://docs.spring.io/spring-boot/docs/2.7.x/reference/html/executable-jar.html#appendix.executable-jar) will be discussed here.
+
+!!! important "Chutney rapid technical description"
+    Chutney server is a Spring Boot application running with [Undertow](https://undertow.io/) using existing Spring stack (mvc, webflux, security).
+
+    Chutney UI is an [Angular](https://angular.io/) web application.
+
+    **Chutney follows Angular and Spring Boot last versions and associated dependencies.**
+
+!!! note "Packaging example"
+    The module [local-dev](https://github.com/chutney-testing/chutney/tree/master/packaging/local-dev) is such a packaging.
```

Comment:
Maven module local-dev shows one way of packaging Chutney.
Use it as an example to make your own package.